cspell.json

```
@@ -34,6 +34,7 @@
 		"tsup",
 		"Unstaged",
 		"vitest",
-		"wontfix"
+		"wontfix",
+		"falsey"
```

[Typo] MDN lists it as `falsy` 😄 https://developer.mozilla.org/en-US/docs/Glossary/Falsy. I'd say let's stick with that?

```suggestion
		"wontfix"
```